fixtures/chauvet-professional/vesuvio-rgba.json

```diff
@@ -0,0 +1,699 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Vesuvio RGBA",
+  "shortName": "Chauvet Jet",
+  "categories": ["Smoke", "Color Changer"],
+  "meta": {
+    "authors": ["Beerady", "Luke Nelson"],
+    "createDate": "2021-09-09",
+    "lastModifyDate": "2022-10-17"
+  },
+  "links": {
+    "productPage": [
+      "http://web.archive.org/web/20160421173724/http://www.chauvetprofessional.com/products/vesuvio-rgba/"
+    ],
+    "manual": [
+      "https://www.chauvetprofessional.com/wp-content/uploads/2015/06/Vesuvio-RGBA_UM_Rev4_WO.pdf"
+    ],
+    "video": ["https://www.youtube.com/watch?v=HH5G92l4USw"]
+  },
+  "rdm": {
+    "modelId": 37266,
+    "softwareVersion": "2016"
+  },
+  "physical": {
+    "dimensions": [336, 488, 231],
+    "weight": 13.6,
+    "power": 1590,
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Fog": {
+      "capability": {
+        "type": "Fog",
+        "fogType": "Fog"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightness": "off"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "brightness": "off"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "brightness": "off"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "brightness": "off"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber",
+        "brightness": "off"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speed": "0Hz"
+      }
+    },
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 11],
+          "type": "ColorPreset",
+          "comment": "Amber/Blue"
+        },
+        {
+          "dmxRange": [12, 12],
+          "type": "ColorPreset",
+          "comment": "Amber"
+        },
+        {
+          "dmxRange": [13, 15],
+          "type": "ColorPreset",
+          "comment": "Amber/Blue dim to bright"
+        },
+        { "dmxRange": [16, 16], "type": "ColorPreset", "comment": "Green" },
```

Comment:
Could you please configure your formatter to not delete deliberate linebreaks? Having this capability in only one line while all the others span multiple lines is really distracting.

Reply:
Yes, my apologies.